### PR TITLE
Update top-level README.md

### PR DESCRIPTION
**Pull request by ikornienko**
Update README file to give slightly more directions to the users who land into the repo by following the product documentation link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
-# helm-charts
-Helm charts for StackRox Kubernetes Security Platform
+# Helm charts for the StackRox Kubernetes Security Platform
+
+To use Helm charts to install the [StackRox Kubernetes Security Platform](https://www.stackrox.com/platform/)
+follow the instructions in the `README.md` file under the corresponding version
```
Comment (clickboo) [inline README.md:4]: StackRox version

Comment (clickboo) [reply README.md:4]: An example would be nice. If you are deploying 3.0.41.4 use the README at <link here> ? @gaurav-nelson thoughts?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,10 @@
-# helm-charts
-Helm charts for StackRox Kubernetes Security Platform
+# Helm charts for the StackRox Kubernetes Security Platform
+
+To use Helm charts to install the [StackRox Kubernetes Security Platform](https://www.stackrox.com/platform/)
```
Comment (clickboo) [inline README.md:3]: To use Helm charts to install Sensor, Collector and Admission controller components of the StackRox Kubernetes Security Platform ... [cc: @gaurav-nelson ]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,10 @@
...
+follow the instructions in the `README.md` file under the corresponding version
+directory.
+
```
Comment (gaurav-nelson) [inline README.md:6]:
```suggestion
After you [install StackRox Central](https://help.stackrox.com/docs/get-started/quick-start/#install-stackrox-central),
you can use helm charts to install Sensor, Collector, and Admission Controller. 

> **IMPORTANT**
>
> We publish new Helm charts with every new release of the StackRox
> Kubernetes Security Platform. Make sure to use a version that matches the
> version of the StackRox Kubernetes Security Platform you've installed.

For example, to install  Sensor, Collector, and Admission Controller using Helm
charts for the StackRox Kubernetes Security Platform version 3.0.41.4, view the
instructions in the
[`3.0.41.4` directory](https://github.com/stackrox/helm-charts/tree/master/3.0.41.4).

```

File: README.md (hunk 1 above)
Comment (gaurav-nelson) [reply README.md:4]: Thanks @boo, I have included this in my suggestion.

File: README.md (hunk 3 above)
Comment (clickboo) [reply README.md:6]: link to the README instead of the directory in which it exists? ^ @gaurav-nelson

Comment (gaurav-nelson) [reply README.md:6]: Thanks @clickboo 

I was thinking that the README always shows up when we open the directory on GitHub, but yes we can directly link to README as well.

Comment (ikornienko) [reply README.md]: It makes to link the directory, as it's where user will start exploring stuff if they're using GitHub web UI. And GitHub web UI will immediately show README per Gaurav's point. So let's keep it this way.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,20 @@
-# helm-charts
-Helm charts for StackRox Kubernetes Security Platform
+# Helm charts for the StackRox Kubernetes Security Platform
+
+After you [install StackRox Central](https://help.stackrox.com/docs/get-started/quick-start/#install-stackrox-central),
+you can use helm charts to install Sensor, Collector, and Admission Controller. 
+
+> **IMPORTANT**
+>
+> We publish new Helm charts with every new release of the StackRox
```
Comment (clickboo) [inline README.md:8]: minor edit: remove the word "new"

**Pull request merged**